website/cue/reference/components/transforms/reduce.cue

```diff
@@ -120,12 +120,16 @@ components: transforms: reduce: {
 						type: string: {
 							enum: {
 								array:          "Each value is appended to an array."
+								longest:        "Retains the longest array seen"
+								shortest:       "Retains the shortest array seen"
```

Comment:
This may be a bit pedantic, but could we call these `longest_array` and `shortest_array`? I could also see people thinking they could be used with strings otherwise.